From cf3b0bbf50382ae75989f0144bebf43b2ed694a9 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Wed, 23 Jun 2021 18:21:25 +0200
Subject: [PATCH] Always build kirigami plugin

The style uses separatorColor which is defined in the kirigami plugin. Therefore the style requires the plugin and we need to ensure that it's always built
---
 CMakeLists.txt | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f131a3d..73602cd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,11 +38,7 @@ include(KDEGitCommitHooks)
 #TODO: Clean up dependencies. I was experimenting.
 find_package(Qt5 ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE COMPONENTS Core Gui Qml Quick QuickControls2 QuickTemplates2 DBus)
 
-find_package(KF5 ${KF5_DEP_VERSION} REQUIRED COMPONENTS GuiAddons Config Kirigami2 CoreAddons)
-
-# IconThemes and ConfigWidgets are optional
-find_package(KF5IconThemes ${KF5_DEP_VERSION})
-find_package(KF5ConfigWidgets ${KF5_DEP_VERSION})
+find_package(KF5 ${KF5_DEP_VERSION} REQUIRED COMPONENTS GuiAddons Config Kirigami2 CoreAddons ConfigWidgets IconThemes)
 
 if (NOT APPLE AND NOT WIN32)
     find_package(X11)
@@ -95,9 +91,7 @@ add_definitions(-DKF_DISABLE_DEPRECATED_BEFORE_AND_AT=0x054900)
 
 add_subdirectory(style)
 
-if (KF5IconThemes_FOUND AND KF5WidgetsAddons_FOUND)
 add_subdirectory(kirigami-plasmadesktop-integration)
-endif(KF5IconThemes_FOUND AND KF5WidgetsAddons_FOUND)
 
 # FIXME: this assumes there will always be a QQC2 release for each Qt release, needs
 # something to discover what's the latest qqc2 version available
